bokeh/core/properties.py

```diff
@@ -1181,7 +1187,9 @@ def _sphinx_type(self):
     def from_json(self, json, models=None):
         if json is None:
             return None
-        elif isinstance(json, list):
+        elif isinstance(json, bokeh_seq_types):
+            if isinstance(self.item_type, Any):
+                return json
```

Comment:
> `if isinstance(self.item_type, Any)`

btw. If only inheritance structure was set properly up in `bokeh.core.properties`, this could would be equivalent of `if True`. This is an equivalent of asking if `isinstance(value, object)` for some `value`.